src/main/resources/logback.xml

```diff
@@ -6,7 +6,11 @@
             <pattern>[%date] %level %message \(%logger:%thread\)%n</pattern>
         </encoder>
     </appender>
+    <appender name="async" class="ch.qos.logback.classic.AsyncAppender">
```

Comment:
Do we flush this on termination?

http://logback.qos.ch/manual/appenders.html#AsyncAppender
http://logback.qos.ch/manual/configuration.html#stopContext